plugins/source/okta/client/spec.go

```diff
@@ -0,0 +1,6 @@
+package client
+
+type Spec struct {
+	Token  string `yaml:"token,omitempty"`
```

Comment:
it should be json (we use json everywhere apart from the first step in the CLI where we read yaml and then convert to json - just like in k8s)